cassdegrees/templates/staff/creation/createprogram.html

```diff
@@ -127,6 +127,13 @@
                         "to students or be selectable as a requirement elsewhere.\n\nAre you sure you want to continue?")) {
                     return false;
                 }
+                // If the user has chosen to publish a plan but the unit count is inconsistent, verify with the user
+                if (document.getElementById("id_publish").checked &&
+                    !isValidUnitCount(parseInt(document.getElementById("id_units").value)) &&
+                    !confirm("You have created a plan that contains a different number of units than you have " +
```

Comment:
Not sure how exact this phrasing is - "different number of units than you have specified" could read better - perhaps "a different number from the number of units you specified for the entire plan"?